doc/for-beginners.md

```diff
@@ -0,0 +1,80 @@
+# Enable full `rust-analyzer` support of `cg_gcc` for your editor
+
+## Steps to Follow
+1. (Already done in by project.) Add the following lines to `Cargo.toml`
+```toml
+[package.metadata.rust-analyzer]
+rustc_private = true
+```
+2. Install rust-src with rustup: `rustup component add rust-src`.
+   NOTE: Make sure you've switched to the corresponding toolchain as is used by cg_gcc
+3. Set the lsp option `rust-analyzer.rustc.source = "discover"` (you can also set it to an explicit path), depending on your editor.
+    1. Neovim(nvim-lspconfig):
```

Comment:
Please drop this part, there are a lot of IDEs, adding docs for some of them is not useful.

Reply:
I wrote this part for  [this comment.](https://github.com/rust-lang/rustc_codegen_gcc/issues/428#issuecomment-1949630003)

Reply:
In my point of view, it may be trivial for us that these steps are necessary, but probably harder for others to figure it out. Adding this may potentially add to contributors(since we only have no more than 100 developers).